doc/api/worker_threads.md

```diff
@@ -518,6 +521,8 @@ changes:
   If `options.eval` is `true`, this is a string containing JavaScript code
   rather than a path.
 * `options` {Object}
+  * `args` {string[]} List of string arguments which would be appended to
```

Comment:
I think it makes sense to call this `argv` for consistency?

Reply:
At first glance I'd thought it could be `argv` too. Yet after considering *the `argv`* would be appended to `[execPath, __filename]`, but not the actual `process.argv`, it might be confusing to name it as `argv`.

Reply:
Renamed to `argv`. PTAL :)